ios/App/App/RegobsPlugin.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  RegobsPlugin.swift
+//  App
+//
+//  Created by Jørgen Loe Kvalberg on 29/04/2022.
+//
+//  The @objc decorators are required to make sure Capacitor’s runtime
+//  (which must use Objective-C for dynamic plugin support) can see it.
+
+import Foundation
+import Capacitor
+
+
+// Whenever you add or remove methods in RegobsPlugin.swift, RegobsPlugin.m must be updated.
+@objc(RegobsPlugin)
+public class RegobsPlugin: CAPPlugin {
+    
+    private let regobs = Regobs()
+    
+    @objc func echo(_ call: CAPPluginCall) {
```

Comment:
Etter vi er blitt enige om interface, så kan jeg lage riktige metoder for iOS, men at det kastes en slags "NotImplemtedError", så blir det litt ryddigere.

Reply:
Har forsøkt å endre ios-pluginen nå og stubber de nye funksjonene.